packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.puppeteer.test.js

```diff
@@ -201,11 +201,13 @@ describe('Checkboxes', () => {
       describe('with conditional items with special characters', () => {
         it('does not error when ID of revealed content contains special characters', async () => {
           // Errors logged to the console will cause this test to fail
-          await render(
-            page,
-            'checkboxes',
-            examples['with conditional items with special characters']
-          )
+          return expect(
```

Comment:
**nitpick** If we no longer `await` inside a function and `return` the promise from `expect`, the test function no longer needs to be `async` and can just be a plain function.

Reply:
Looks like there's an ESLint rule for that: https://eslint.org/docs/latest/rules/require-await. It can probably be enabled as its own PR, though.

Reply:
From a quick test it looks like it does flag this and another instance on line 75. (57 across the entire repo).

Happy for that to be a follow up thing?

Reply:
Yup, I thought there'd be more across the repo, so let's not worry about it for this PR and raise a separate one for `require-await`.